index.html

```diff
@@ -4732,6 +4717,28 @@ <h2>Equivalence Properties</h2>
       </p>
 
     </section>
+
+    <section>
+      <h2>
+Content Integrity Protection
+      </h2>
+      <p>
+DID documents which include external JSON-LD contexts (see
+<a href="#json-ld"></a>) or any other links to external machine-readable
+content are vulnerable to tampering.
+      </p>
+      <p>
+DID document consumers can cache local static copies of JSON-LD contexts
+and/or verify the integrity of external contexts against the cryptographic
+hash for the context as registered in the DID specification registries
+(see the registration process for more detail) [[?DID-SPEC-REGISTRIES]].
+      </p>
+      <p>
+Implementers can also consider other means to protect links to external
+content as best suited to their application. Examples of URL schemes which
+enforce content integrity are [[HASHLINK]] and [[IPFS]].
```

Comment:
providing 2 options here and a different option in the registry is terrible. either describe one solution in 1 place. or remove this language.

Reply:
Agree, removing the language as I doubt @rhiaro (or anyone else) would object.